packages/govuk-frontend/src/govuk/components/button/button.yaml

```diff
@@ -2,15 +2,15 @@ params:
   - name: element
     type: string
     required: false
-    description: Whether to use an `input`, `button` or `a` element to create the button. In most cases you will not need to set this as it will be configured automatically if you use `href` or `html`.
+    description: Whether to use an `input`, `button` or `a` element to create the button component. In most cases you will not need to set this as it will be configured automatically if you use `href` or `html`.
   - name: text
     type: string
     required: true
-    description: If `html` is set, this is not required. Text for the button or link. If `html` is provided, the `text` option will be ignored and `element` will be automatically set to `button` unless `href` is also set, or it has already been defined. This option has no effect if `element` is set to `input`.
+    description: If `html` is set, this is not required. Text for the `input`, `button` or `a` elements. If `html` is provided, the `text` option will be ignored and `element` will be automatically set to `button` unless `href` is also set, or it has already been defined.
```

Comment:
`element` singular? Since it's only one of those three?

Reply:
I wasn't totally sure as we have a mix of:

* "For input and button **elements**"
* "…no effect on a or input **elements**"

But sounds better where "for the…" is used now